src/wantlist/index.js

```diff
@@ -51,8 +55,9 @@ class Wantlist {
     return new Map(Array.from(this.set.entries()).sort())
   }
 
-  contains (key) {
-    return this.set.get(mh.toB58String(key))
+  contains (cid) {
+    const cidStr = cid.toBaseEncodedString()
+    return !!this.set.get(cidStr)
```

Comment:
I'm okay with refactoring it so that it's not needed anymore, but there are places where changing this breaks things, that's all I wanted to point out.